index.html

```diff
@@ -5,31 +5,32 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
+  <!-- TO BE DELETED
+  <link rel="stylesheet" href="construction.css"> -->
+  <script src=""></script>
   <title>Portfolio || Javier</title>
 </head>
 <body>
-  <header class="bg-white fill-100">
-    <nav class="xp5vh m5vh flex navigation">
+  <header class="bg-white fill-100 ">
+    <nav class="xp5vh m5vh flex navigation" id="portfolio">
       <span>
         <a href="https://">
           <img class="logo" src="assets/images/My Logo.svg" alt="Logo">
         </a>
       </span>
-      <span>
-        <a class="mv" href="http://">
-          <img class="menu" src="assets/images/menu-template.svg" alt="Navigation menu">
-        </a>
+      <span class="mv" id="mobile-menu">
+        <img class="menu btn-menu" src="assets/images/menu-template.svg" alt="Navigation menu">
       </span>
       <span class="dv">
         <ul class="g20px xul">
-          <li class="link-text"><a href="http://">Portfolio</a></li>
-          <li class="link-text"><a href="http://">About</a></li>
-          <li class="link-text"><a href="http://">Contact</a></li>
+          <li class="link-text"><a href="#portfolio">Portfolio</a></li>
+          <li class="link-text"><a href="#about">About</a></li>
```

Comment:
- [ ] When users click on the About option, The about section of the page should be displayed but it's displaying the hero section. Please make sure that when users click on the About option the below part of the page should be displayed.

![image](https://user-images.githubusercontent.com/83803414/208695444-e32eb2a9-fc65-462c-aa8c-af7a42e2e0af.png)